Sources/KituraNet/HTTP/HTTPRequestHandler.swift

```diff
@@ -93,7 +93,7 @@ internal class HTTPRequestHandler: ChannelInboundHandler, RemovableChannelHandle
                     } catch {
                         Log.error("Failed to send error response")
                     }
-                    context.close()
+                    _ = context.close()
```

Comment:
```suggestion
                    context.close(promise: nil)
```

is the correct (and faster (saves an allocation)) way to say "I don't care about the result"

Reply:
Thanks!